doc/man/bitcoin-cli.1

```diff
@@ -22,6 +22,10 @@ This help message
 .IP
 Specify configuration file (default: bitcoin.conf)
 .HP
+\fB\-netconf=\fR<file>
```

Comment:
It could be useful if you wanted to temporarily restart bitcoind without using the network.conf file. You could have a bitcoin-alt.conf that specified netconf=network-alt.conf so that you could load bitcoind with -conf=bitcoin-alt.conf and have that load network specific alternative configurations too. Not sure how realistic/valuable those are, though. Not having the option would be inconsistent; and having the option makes the name of the config file at least a little discoverable...